webapp/src/components/Navbar.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button, Grid, IconButton } from '@mui/material';
-import EventNoteIcon from '@mui/icons-material/EventNote';
 import React, { useContext } from 'react';
-
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import PeopleIcon from '@mui/icons-material/People';
```

Comment:
Please add a single blank line after PeopleIcon. It'll be a separator between imported things from packages and imported things that created by team.

Reply:
Hi Veranika, we added the blank line! :)